ml_peg/app/utils/load.py

```diff
@@ -57,11 +67,20 @@ def rebuild_table(filename: str | Path, id: str) -> DataTable:
         elif column.get("type") == "numeric" or is_numeric_column(data, column_id):
             column["type"] = "numeric"
             column.setdefault("format", sig_fig_format())
+
+        base_name = column_name if isinstance(column_name, str) else None
```

Comment:
In what situations are column_name/column_id not strings, or at minimum not able to be converted into strings (e.g. if for some reason they were numerical)?

Robustness is good, but maybe we should just raise an error, as otherwise the logic ends up more and more complex.